packages/client/hmi-client/src/components/widgets/pdf-embed.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+	<div id="adobe-dc-view"></div>
```

Comment:
Does having an `id=xyz` hardwired like this cause a problem if we have multiple pdf tabs open at the same time? or the case where they are opened across multiple panels/windows?

Reply:
@shawnyama can you look into it please.

Reply:
I think it's fine. One document component is shown at a time so there aren't going to be duplicate ids